src/dlgProfilePreferences.cpp

```diff
@@ -1626,6 +1606,19 @@ void dlgProfilePreferences::copyMap()
     mudlet::self()->setShowMapAuditErrors(savedOldAuditErrorsToConsoleEnabledSetting);
 }
 
+void dlgProfilePreferences::setLogDir()
```

Comment:
We (well, I) do try to name SIGNAL/SLOT functions with a `signal_`/`slot_` prefix as it helps to remind some of us coders that they are signal/slot functions (and thus are expected to be *wired-up* in a `connect(`...`)` - and if they are not so used then they do not need to be declared as such - and they can be demoted out of the scope of the `moc` {remember to rerun `qmake` on the project when ever a signal/slot is added/removed from a class})! :neckbeard:

Reply:
I see! I'll modify the name of the function.